### DISCUSSION TITLE
Google Style Missing

### DISCUSSION THREAD

**Discussion by Andy16823** [cefsharp/CefSharp#4788] [General]
Hello, for some reasons the css from google is missing in my browser.

![image](https://github.com/cefsharp/CefSharp/assets/6460466/e53391c9-fea1-4374-ac61-04fe80ca7c7b)

i think its something with my settings.

**Comment by amaitland** [comment]
How does the https://github.com/cefsharp/CefSharp.MinimalExample behave?

Google loads as expected for me.

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

**Comment by Andy16823** [comment]
The issue was with the custom user agent I tried to use. Without my custom user agent, it's working now.
